index.html

```diff
@@ -0,0 +1,197 @@
+</section>
+<!--Card works 3 ends-->
+
+<!--Card works 4 starts-->
+<section class="works-card">
+    <img src="./assets/Snapshoot Portfolio (3).png" alt="Recent Work">
+    <h2 class="works-card-heading-large">Multi-Post<br>Stories</h2>
+    <div class="works-card-client">
+        <p class="paragraph">CANOPY</p>
+        <p class="fa-solid fa-circle works-card-client-counter"></p>
+        <p class="works-card-client-role paragraph">Back End Dev</p>
+        <p class="fa-solid fa-circle works-card-client-counter"></p>
+        <p class="works-card-client-year paragraph">2015</p>
+    </div>
+    <p class="paragraph works-card-paragraph">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="works-card-tags">
+        <li class="works-card-tag">html</li>
+        <li class="works-card-tag">css</li>
+        <li class="works-card-tag">javascript</li>
+    </ul>
+    <div>
+    <button>See Project</button>
+    </div>
+</section>
+<!--Card works 4 ends-->
+
+<!-- About Section Starts Here -->
+<section class="about">
+    <div class="headline">
+        <h2 class="headline-heading heading-large">About <br> Myself</h2>
+        <p class="headline-paragraph paragraph">I’m a software developer! I can help you build a product , feature
+            or website
+            Look
+            through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+            me.
+        </p>
+        <p class="headline-sm-suport">LET’S CONNECT</p>
+        <ul class="social-icons">
+            <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+        </ul>
+        <button>Get my resume</button>
+    </div>
+    <div class="about-skills">
+        <div class="about-skills-heading">
+            <h2 class="about-heading-small">Languages</h2>
+            <i class="fa-solid fa-angle-down"></i>
+        </div>
+        <div class="about-skill">
+            <img src="./assets/Ellipse Javascript 1.png" alt="js">
+            <h2 class="about-skill-text">Javascript</h2>
+        </div>
+        <div class="about-skill">
+            <img src="./assets/Ellipse html 1.png" alt="html">
+            <h2 class="about-skill-text">HTML</h2>
+        </div>
+        <div class="about-skill">
+            <img src="./assets/Ellipse css 1.png" alt="css">
+            <h2 class="about-skill-text">CSS</h2>
+        </div>
+        <div class="about-skills-heading">
+            <h2 class="about-heading-small">Frameworks</h2>
+            <i class="fa-solid fa-angle-right"></i>
+        </div>
+        <hr>
+        <div class="about-skills-heading">
+            <h2 class="about-heading-small">Skills</h2>
+            <i class="fa-solid fa-angle-right"></i>
+        </div>
```

Comment:
- [ ] Good job creating all the required elements to match with the Figma template :+1:, but for semantics, the technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul` and each of them in a `li` tag. And also the technology in each category(e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li` :100:

Reply:
I updated it as mentioned using ul and li tags.